packages/client/src/modules/user-profile/components/user-profile-card/user-profile-card.component.html

```diff
@@ -38,7 +38,7 @@
                     <i class="fas fa-chevron-right"></i>
                 </span>
                 <button class="user-profile__main__swipe__button" (click)="swipe(true)">
-                    <i class="user-profile__main__swipe__icon fas fa-heart" [style.scale]="likeScale"></i>
+                    <i class="user-profile__main__swipe__icon fas fa-thumbs-up" [style.scale]="likeScale"></i>
```

Comment:
This icon is there two times. It also appears when the card is opened. Now it is still a hearth 
<img width="89" alt="Screenshot 2024-02-22 at 16 39 11" src="https://github.com/ProjetAGIR/projet-agir/assets/20962021/54323828-5eb3-4b0f-b051-ecd492ba03eb">